thermo_nerf/render/renderer.py

```diff
@@ -19,6 +19,7 @@ class RenderedImageModality(Enum):
     depth = "depth"
     accumulation = "accumulation"
     thermal = "thermal"
+    thermal_combined = "thermal_combined"
```

Comment:
Can you make all those uppercase to follow the standard?

Move it to its own file in `thermo_nerf` so you can use it elsewhere